site/src/sphinx/setup-configuration.rst

```diff
@@ -147,7 +147,7 @@ Core properties
   - ``method`` (string)
 
     - the replication method. ``NONE`` indicates 'standalone mode' without replication. ZooKeeper-based
-      multi-master replication will be explained later in this page.
+      multi-master replication will be explained in :ref:`replication`.
```

Comment:
Oh, thanks! How about then

    See :ref:`replication` to learn how to configure ZooKeeper-based multi-master replication.